src/services/redis/leaderboardCache.ts

```
@@ -0,0 +1,299 @@
+}
+
+/**
+ * Invalidate leaderboard cache (for data updates)
+ */
+export async function invalidateLeaderboardCache(
+  sortBy?: "attendance" | "midmarks",
+  filters?: {
+    year?: string;
+    branch?: string;
+    section?: string;
+  }
```

ESLint

[error] Insert `,`

(prettier/prettier)